inst/templates/ad_adlb.R

```diff
@@ -0,0 +1,331 @@
+# Name: ADLB
+#
+# Label: Lab Analysis Dataset
+#
+# Input: adsl, lb
+library(admiral)
+library(admiraltest) # Contains example datasets from the CDISC pilot project
+library(dplyr)
+library(lubridate)
+library(stringr)
+
+# ---- Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("lb")
```

Comment:
Replace `data("lb")` with `data(admiral_lb)` then just after his put `lb <- admiral_lb` and `adsl <- admiral_adsl`. Basically like it is in ad_advs script.